Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/KeySubmissionsCard.kt

```diff
@@ -22,7 +24,18 @@ class KeySubmissionsCard(parent: ViewGroup) :
     override val onBindData: HomeStatisticsCardsKeysubmissionsLayoutBinding.(
         item: StatisticsCardItem,
         payloads: List<Any>
-    ) -> Unit = { item, payloads ->
-        item.stats as KeySubmissionsStats
+    ) -> Unit = { item, _ ->
+        with(item.stats as KeySubmissionsStats) {
+            viewBinding.value.keySubmissionsLabel = getPrimaryLabel(context)
+
+            viewBinding.value.keySubmissions =
+                formatStatisticalValue(context, keySubmissions.value, keySubmissions.decimals)
+            viewBinding.value.sevenDayAverage =
+                formatStatisticalValue(context, sevenDayAverage.value, sevenDayAverage.decimals)
+            viewBinding.value.total =
+                formatStatisticalValue(context, total.value, total.decimals)
+
```

Comment:
You could add this click listener. Then the info will show up after Alex has merged the info feature. 

```suggestion
        statisticsCardInfoButton.setOnClickListener {
            item.onHelpAction.invoke(item.stats)
        }
```

Reply:
yes, I added it also for `InfectionsCard`